app/renderer/components/main/findbar.js

```diff
@@ -240,58 +247,146 @@ class FindBar extends React.Component {
       }
     }
 
-    return <div className='findBar' style={findBarStyle} onBlur={this.onBlur}>
-      <div className='searchContainer'>
-        <div className='searchStringContainer'>
-          <span className='searchStringContainerIcon fa fa-search' />
-          <input
-            ref={(node) => { this.searchInput = node }}
-            type='text'
-            spellCheck='false'
-            onContextMenu={this.onContextMenu}
-            onFocus={this.onInputFocus}
-            onKeyDown={this.onKeyDown}
-            onInput={this.onInput}
-          />
-          <span
-            className='searchStringContainerIcon fa fa-times findClear'
-            onClick={this.onClear}
-          />
-        </div>
-        <span className='findMatchText'>{this.findTextMatch}</span>
-        <BrowserButton
-          iconOnly
-          iconClass={globalStyles.appIcons.findPrev}
-          inlineStyles={findBarStyle}
-          testId='findBarPrevButton'
-          disabled={this.props.numberOfMatches <= 0}
-          onClick={this.onFindPrev}
+    // See autofillAddressPanel.js
+    // Ref: https://github.com/brave/browser-laptop/pull/7164#discussion_r100586892
```

Comment:
@petemill do you think this applies to this case?

Reply:
@luixxiul if you are asking will the ref={ ... } work here - then yes because it's directly on html element.